### PR TITLE
Pw 18 - Add skeleton wireframes for all component pages

### PR DESCRIPTION
**Pull request by neonseal**

### CODE REVIEW

File: src/components/ConnectPage/ConnectPage.module.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    border-color: white; // Remove when content is added
+    margin: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 400px;
+    width: var(--main-width);
+
+    .profileImgColumn {
+        // padding: 10px;
```
Comment (WhiterDig) [inline src/components/ConnectPage/ConnectPage.module.scss]: Is there a reason we left this here?

**Review by WhiterDig — COMMENTED** [review]
Everything looks good to me. Nice and clean, only had a question about the one spot you left in some commented out css.

File: src/components/ConnectPage/ConnectPage.module.scss (hunk 1 above)
Comment (neonseal) [reply src/components/ConnectPage/ConnectPage.module.scss]: Nope! Just forgot to remove it. I'll fix that now

Comment (neonseal) [reply src/components/ConnectPage/ConnectPage.module.scss]: Updated

**Review by WhiterDig — APPROVED** [review]

**Pull request merged**